lib/util/miq_apache/miq_apache.rb

```diff
@@ -125,7 +125,7 @@ def self.status(full = true)
     end
 
     def self.version
```

Comment:
@brandondunne 
The only place I see it is in lib/spec/util/miq_apache/control_spec.rb